src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7403,4 +7400,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureLambdaParamsArray" xml:space="preserve">
     <value>lambda params array</value>
   </data>
+  <data name="WRN_ParamsArrayInLambdaOnly" xml:space="preserve">
+    <value>Parameter {0} has params modifier in lambda but not in target delegate type.</value>
+  </data>
+  <data name="WRN_ParamsArrayInLambdaOnly_Title" xml:space="preserve">
+    <value>Parameter {0} has params modifier in lambda but not in target delegate type.</value>
```

Comment:
>{0} [](http://example.com/codeflow?start=21&length=3)

The placeholder should be removed since the title is used without arguments: "Parameter has params modifier ..."

Reply:
Oh, good catch, I was too eager. Fixed: 3bafef166ead2f9291a7c7b147cbef5777e9ebad